docs/bagit/readme.txt

```diff
@@ -0,0 +1,51 @@
+This zip file contains a HydroShare resource in Bagit format.  
```

Comment:
on the top under resource id directory, along with manifest files and bagit.txt, which conforms to bagit spec according to @horsburgh